src/relayclient/DevRelayClient.ts

```diff
@@ -225,7 +226,7 @@ export class DevRelayClient extends RelayClient {
     const relayInfo = await waitForRelay(relayServer.url as string + '/getaddr', 5000, (res) => {
       if (res?.data?.Ready === true) {
         return {
-          ...res.data,
+          relayManager: res.data.RelayServerAddress,
           pctRelayFee: '0',
```

Comment:
why add fees here? they never returned by a relay getaddr

Reply:
What do you mean? This is your code that adds fees, not mine. I convert ```RelayServerAddress``` to ```relayManager``` so that response confirms to the expected EventInfo interface